<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frm_Main</class>
 <widget class="QDialog" name="Frm_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HKW工具箱</string>
  </property>
  <property name="windowIcon">
   <iconset resource="HKW-Tools.qrc">
    <normaloff>:/Icon/Icon.png</normaloff>:/Icon/Icon.png</iconset>
  </property>
  <widget class="QGroupBox" name="funs_GroupBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>111</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>功能列表</string>
   </property>
   <widget class="QListWidget" name="funs_ListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>91</width>
      <height>411</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="devicesInfos_GroupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>设备选项</string>
   </property>
   <widget class="QComboBox" name="syncDevices_ComboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="showNumOfDevices_Lable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>当前连接了 0 个设备</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="clickTo_Open_Frm_ConnectDevice_PushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
     </font>
    </property>
    <property name="text">
     <string>连接&amp;&amp;配对设备</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="deviceInfo_GroupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>181</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>设备信息</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>型号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU型号:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>生产商:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>安卓版本:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前Dpi:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前分辨率:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>储存空间:</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="show_UsedStorage_ProgressBar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_DeviceModel_LineEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_CpuModel_LineEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>50</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_Manufacturer_LineEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>80</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_AndroidVersion_LineEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_CurrentDpi_LineEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>140</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_CurrentResolution_LineEdit">
     <property name="geometry">
      <rect>
       <x>82</x>
       <y>170</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>总共:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>240</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>已用:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_StorageSize_LineEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>240</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="show_UsedStorage_LineEdit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>240</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clickTo_ReloadDeviceInfo_PushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>141</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>手动刷新</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="HKW-Tools.qrc"/>
 </resources>
 <connections/>
</ui>
